CONTRIBUTING.md

```diff
@@ -133,25 +128,17 @@ Further paragraphs come after blank lines.
 ```
 
 ### 2.5 Submit for Review
-To make sure that you don't stumble upon any irregularity while submitting your changes for review, the origin must be renamed to 'gerrit'.
+To make sure that you don't stumble upon any irregularity while submitting your changes for review, the remote fork must be added.
 Perform the following command from the root of glusto-tests:
 
 ```
-$ git remote rename origin gerrit
-```
-
-The gerrit review process used for glusto-tests is by 'git-review'. You must have git-review installed in order to submit a patch for review.
-
-Install git-review using pip:
-
-```
-$ pip install git-review
+$ git remote add fork https://github.com/<your-github-username>/glusto-tests
 ```
 
-Once installed, you simply need to provide the review command:
+You simply need to provide the review command:
 
 ```
-$ git-review
+$ git push fork hellopatch
```

Comment:
git push -u origin new_branchname

Reply:
Pushing directly to origin isn't a good habit.